### PR TITLE
Add openfst

### PR DESCRIPTION
**Pull request by 183amir**
closes #2569 (replacement)

### CODE REVIEW

**Comment by 183amir** [comment]
@mcernak I fixed the issues.
@conda-forge/staged-recipes can I get a review on this pull request please?

File: recipes/openfst/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# we need to run autoreconf and automake here because somehow `make` thinks
+# that it needs to update the scripts. This answer may be relevant:
+# https://stackoverflow.com/a/30386141/1286165
+autoreconf
+bash ./configure --prefix=${PREFIX} --enable-static=no
+automake
+make -j ${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/openfst/build.sh]: Is there a `make check` we can run here?

File: recipes/openfst/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: openfst
+  version: "1.6.1"
```
Comment (ocefpaf) [inline recipes/openfst/meta.yaml]: Please use a jinja variable for the version to avoid problems with different number in the future.

**Comment by 183amir** [comment]
Done @ocefpaf

**Pull request merged**